protocol/SignalingProtocol.proto

```diff
@@ -348,4 +348,15 @@ message SdkTranscriptEvent {
 
 message SdkTranscriptFrame {
     repeated SdkTranscriptEvent events = 1;
-}
\ No newline at end of file
+}
+
+message SdkRemoteVideoUpdateFrame {
+    repeated SdkVideoSubscriptionConfiguration added_or_updated_video_subscriptions = 1;
+    repeated string removed_video_subscription_mids = 2;
+}
+
+message SdkVideoSubscriptionConfiguration {
+   required string mid = 1;
```

Comment:
Can we add a comment about what a mid is?

Reply:
sure, did so in `SignalingClientVideoSubscriptionConfiguration`